xwiki-commons-core/xwiki-commons-job/src/main/java/org/xwiki/job/AbstractJob.java

```diff
@@ -288,7 +288,7 @@ protected void jobFinished(Throwable error)
                 if (this.request.getId() != null) {
                     this.store.storeAsync(this.status);
                 }
-            } catch (Throwable t) {
+            } catch (Exception t) {
```

Comment:
Actually this one should not be catched at all I think. I guess it's a leftover.